src/main/java/org/apache/commons/io/FileUtils.java

```diff
@@ -1838,6 +1841,36 @@ public static boolean isSymlink(final File file) {
         return file != null ? Files.isSymbolicLink(file.toPath()) : false;
     }
 
+    /**
+     * Tests whether the specified file is an actual file.
+     * <p>
+     * This method delegates to {@link Files#isRegularFile(Path, LinkOption...)}
+     * </p>
+     *
+     * @param file the file to test.
+     * @return true if the file is a regular file, see {@link Files#isRegularFile(Path, LinkOption...)}.
+     * @since 2.9
+     * @see Files#isRegularFile(Path, LinkOption...)
+     */
+    public static boolean isRegularFile(final File file) {
```

Comment:
If you want to do this kind of rework, add "primitive" methods to PathUtils and then have this method just pass the call forward with the File#toPath() result. The PathUtils method then just becomes a null-safe version of the underlying JRE Files API.